### PR TITLE
Structure README File

### PR DESCRIPTION
**Pull request by SNikhill**
# Description
With this merge, the structure of the README file will be improved. This will help future contributors to have a pleasant experience while seeking out information such as **Setting Up the Project**, **Building the Files**.

### Other Suggestions
While we are at the topic of structuring, I propose to create a `.gitattribute` file to prevent diff pollution caused by varying End Of Lines.

Also, why exactly don't you keep a project specific `package.json` file?

### CODE REVIEW

**Review by juliendonck — DISMISSED** [review]
Great contribution. Thank you very much

File: README.md [hunk 1]
```diff
@@ -1,23 +1,97 @@
-# The [radicle.xyz](https://radicle.xyz) website.
+# The [radicle.xyz](https://radicle.xyz) website
 
-## Build
+This repository is where the source code for [radicle.xyz](https://radicle.xyz)
+has been stored.
```
Comment (cloudhead) [inline README.md:4]: Can we just change this to `This is the repository of the radicle.xyz website.`?

Comment (cloudhead) [reply README.md:4]: Otherwise LGTM.

Comment (SNikhill) [reply README.md:4]: I actually used the same to add weight to the description.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,97 @@
-# The [radicle.xyz](https://radicle.xyz) website.
+# The [radicle.xyz](https://radicle.xyz) website
 
-## Build
+This repository is where the source code for [radicle.xyz](https://radicle.xyz)
```
Comment (juliendonck) [inline README.md:3]:
```suggestion
This is the repository of the radicle.xyz website.
```

File: README.md (hunk 1 above)
Comment (juliendonck) [inline README.md:4]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,96 @@
-# The [radicle.xyz](https://radicle.xyz) website.
+# The [radicle.xyz](https://radicle.xyz) website
 
-## Build
+This is the repository of the radicle.xyz website.
```
Comment (juliendonck) [inline README.md:3]:
```suggestion
This is the repository of the [radicle.xyz](https://radicle.xyz) website.
```

**Review by juliendonck — APPROVED** [review]

**Pull request merged**